src/pybind/mgr/dashboard/module.py

```diff
@@ -525,6 +529,21 @@ def default(self, *args, **kwargs):
                         return None
 
                     if active_uri:
+                        if module.get_module_option('redirect_resolve_ip_addr', False):
```

Comment:
No need to repeat the default, it's taken from the definition above:
```suggestion
                        if module.get_module_option('redirect_resolve_ip_addr'):
```